webrtc.html

```diff
@@ -2871,6 +2929,242 @@
+          </dd>
+
+          <dt>sequence&lt;RTCIceCandidate&gt; getRemoteCandidates()</dt>
+
+          <dd>
+            <p>An array containingr the remote ICE candiates received by
+            this <code><a>RTCIceTransport</a></code>
+            via <code><a href=
+            "#dom-peerconnection-addicecandidate">PeerConnection.addIceCandidate</a></code></p>
+          </dd>
+
+          <dt>RTCIceCandidatePair? getSelectedCandidatePair()</dt>
+          <dd><p>Returns the selected candidate pair on which packets
+          are sent, or NULL if there is no such pair.</p></dd>
+
+          <dt>RTCIceParameters? getLocalParameters()</dt>
+
+          <dd><p>Returns the local ICE parameters received by
+            this <code><a>RTCIceTransport</a></code>
+            via <code><a href=
+            "#dom-peerconnection-setlocaldescription">setLocalDescription</a></code>,
+            or <code>null</code> if the parameters have not yet been
+            received.</p>
+          </dd>
+
+          <dt>RTCIceParameters? getRemoteParameters()</dt>
+
+          <dd><p>Returns the remote ICE parameters received by
+            this <code><a>RTCIceTransport</a></code>
+            via <code><a href=
+            "#dom-peerconnection-setremotedescription">setRemoteDescription</a></code></p>
+            or <code>null</code> if the parameters have not yet been
+            received.</p>
+          </dd>
+
+          <dt>attribute EventHandler onstatechange</dt>
+
+          <dd>
+            This event handler, of event handler event type <code><a href=
+            "#event-icetransport-statechange">statechange</a></code>,
+            MUST be fired any time the <a href=
+            "#dom-icetransport-state"><code><a>RTCIceTransport</a></code> state</a> changes.
+          </dd>
+
+          <dt>attribute EventHandler ongatheringstatechange</dt>
+
+          <dd>
+            This event handler, of event handler event
+            type <code><a href=
+            "#event-icetransport-gatheringstatechange">gatheringstatechange</a></code>,
+            MUST be fired any time the <a href=
+            "#dom-icetransport-gatheringstate"><code><a>RTCIceTransport</a></code>gathering
+            state</a> changes.
+          </dd>
+
+          <dt>attribute EventHandler onselectedcandidatepairchange</dt>
+
+          <dd>
+            This event handler, of event handler event
+            type <code><a href=
+            "#event-icetransport-selectedcandidatepairchange">selectedcandidatepairchange</a></code>,
+            MUST be fired any time
+            the <code><a>RTCIceTransport</a></code>'s selected
+            candidate pair</a> changes.
+          </dd>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCIceParameters">
+          <dt>DOMString usernameFragment</dt>
+          <dd><p>The ICE username fragment as defined in [[!RFC5245]], Section 7.1.2.3.</p></dd>
```

Comment:
This should use [[!ICE]]  since this is what we already use in the rest of the spec (otherwise we'll get duplicate references); this applies also to line 3133